From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Marijn Suijten <marijns95@gmail.com>
Date: Wed, 10 Feb 2021 10:11:44 +0100
Subject: [PATCH] cirrus_sony: Do not unpack unused function pointers from init
 config

These functions are not used; it's rather wasteful readability-wise to
retrieve them.

(The others are not used yet... but will be in future commits)

Signed-off-by: Marijn Suijten <marijns95@gmail.com>
---
 hal/audio_extn/cirrus_sony.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/hal/audio_extn/cirrus_sony.c b/hal/audio_extn/cirrus_sony.c
index 8bb08aa23c137637ecadebe64513a9567470279d..13968a2404813e0e600be1363dcb3892f1766daa 100644
--- a/hal/audio_extn/cirrus_sony.c
+++ b/hal/audio_extn/cirrus_sony.c
@@ -44,8 +44,6 @@ static fp_enable_disable_snd_device_t fp_disable_snd_device;
 static fp_enable_disable_snd_device_t  fp_enable_snd_device;
 static fp_enable_disable_audio_route_t fp_disable_audio_route;
 static fp_enable_disable_audio_route_t fp_enable_audio_route;
-static fp_audio_extn_get_snd_card_split_t fp_audio_extn_get_snd_card_split;
-static fp_platform_get_snd_device_t fp_platform_get_vi_feedback_snd_device;
 
 enum cirrus_playback_state {
     INIT = 0,
@@ -385,9 +383,6 @@ void spkr_prot_init(void *adev, spkr_prot_init_config_t spkr_prot_init_config_va
     fp_enable_snd_device = spkr_prot_init_config_val.fp_enable_snd_device;
     fp_disable_audio_route = spkr_prot_init_config_val.fp_disable_audio_route;
     fp_enable_audio_route = spkr_prot_init_config_val.fp_enable_audio_route;
-    fp_audio_extn_get_snd_card_split = spkr_prot_init_config_val.fp_audio_extn_get_snd_card_split;
-    fp_platform_get_vi_feedback_snd_device = spkr_prot_init_config_val.fp_platform_get_vi_feedback_snd_device;
-
 
     pthread_mutex_init(&handle.fb_prot_mutex, NULL);
 
